conf/layer.conf

```diff
@@ -2,5 +2,5 @@ BBPATH .= ":${LAYERDIR}"
 BBFILES += "${LAYERDIR}/recipes*/*/*.bb ${LAYERDIR}/recipes*/*/*.bbappend"
 BBFILE_COLLECTIONS += "skipjack-hybris-layer"
 BBFILE_PATTERN_skipjack-hybris-layer := "^${LAYERDIR}/"
-BBFILE_PRIORITY_skipjack-hybris-layer = "7"
-LAYERSERIES_COMPAT_skipjack-hybris-layer = "hardknott"
+BBFILE_PRIORITY:skipjack-hybris-layer = "7"
```

Comment:
This might actually still need to be a lower case according to the documentation?
https://www.yoctoproject.org/docs/latest/mega-manual/mega-manual.html#locating-and-parsing-recipes

Reply:
Did that and it gave warning on compile, switching it gave no error.

Reply:
(As it new syntax gave warning, old did not)

Reply:
I tested locally, either appears to work.
Weird that it gave a warning for you tho.
In any case we can change this in the future if it turns out that this was an error or for the other watches if they need the new syntax (Example: https://github.com/AsteroidOS/meta-asteroid/blob/master/conf/layer.conf#L9)

Just to confirm, the `meta-asteroid` did not have this warning?

Reply:
As far as I know, meta-asteroid did not give any warnings, nothing does as of the latest commit